src/events/api/ruleBundles.ts

```diff
@@ -0,0 +1,12 @@
+import { listResource, retrieveResource } from 'common/resources';
+import { IQueryObject } from 'common/utils/queryString';
+import { IRuleBundle } from 'events/models/RuleBundles';
+
+const RULE_BUNDLE_API_URL = '/api/v1/event/rule-bundles';
+
+interface IRuleBundleFilters extends IQueryObject {
+  event?: number;
```

Comment:
Is this the primaryKey for an event?
Can I suggest to change it to `eventId` if so?

Reply:
It is that way for all query parameters in Django Rest Framework, unless you specifically opt to change it every time you create a filer class. It wouldn't really make sense for it to be anything else when it is serialized in the url anyways?

Reply:
Oh, I just meant the key name, not the type! As I found it weird that event is number, while event could also be an IEvent
`eventId?: number` instead of `event?: number`.

Reply:
This maps directly to the query parameters, so I don't really know how we would do that

Reply:
Oh, right, now I understand. Perhaps a comment could be added, something alone the lines of "Primary key of event, serialized and named from API" or something?

Reply:
I think it would potentially be quite tedious to do that for _all_ query parameters which are primary keys?

Could potentially find another way to do query parameters in the future?